frontend/src/components/user-details/components/user-profile/components/avatar-wrapper/avatar-wrapper.tsx

```diff
@@ -1,21 +1,16 @@
 import defaultUserAvatar from 'assets/img/avatar-default.svg';
 import { FC, UserWithPermissions } from 'common/types/types';
 import { Button, Image } from 'components/common/common';
-import {
-  useAppDispatch,
-  useAppSelector,
-  useEffect,
-  useState,
-} from 'hooks/hooks';
+import { useAppDispatch, useAppSelector, useState } from 'hooks/hooks';
 import React from 'react';
 import { userDetailsActions } from 'store/actions';
 
 import styles from './styles.module.scss';
 
 const AvatarWrapper: FC = () => {
-  const { user, userDetails } = useAppSelector((state) => ({
+  const { user, avatarUrl } = useAppSelector((state) => ({
```

Comment:
in the parent component - UserProfile, there is useAppSelector and useAppDispatch, I think there is no need to use it again, you can throw needed info and callbacks through props to AvatarWrapper

Reply:
Agree, we should remove useAppSelector from this component, and pass needed data in props

Reply:
agreed